### ISSUE TITLE
Add support for separate signature registry than the image 

### ISSUE THREAD

**Issue by sambhav** [dlorenc/cosigned#5]
cosign supports uploading signatures to a separate registry than the image itself.

cosigned should be able to support something similar when uploading the public key config map to associate each keys with a registry?

**Comment by dlorenc** [comment]
Sure! We can use the same env car as config somehow.

**Comment by sambhav** [comment]
I might try and contribute this :)

**Comment by dlorenc** [comment]
Sure! Just curious - are you thinking of running this anywhere? I really only put it together as a quick demo/POC, but it appears that some people are actually looking at it seriously :)

**Comment by sambhav** [comment]
If it supports this and #6, definitely. It might be worth considering to promote this repository or something like it to the sigstore project. I believe other people who are using cosign will probably be interested in something like this as well :)

**Comment by dlorenc** [comment]
Yeah that's mainly what I was getting at! We can find some kind of org to move this to if people are really interested in it :)
